From d344b501ab4cbb22305287648bb551c90fbb8ca9 Mon Sep 17 00:00:00 2001
From: Michael Terry <mterry@edx.org>
Date: Tue, 21 Dec 2021 13:53:11 -0500
Subject: [PATCH 0768/1021] fix: add back missing message translation string
 (#784)

It was accidentally removed when we switched to the external
header (frontend-component-header), but the string is still
actually used.
---
 src/generic/messages.js | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/generic/messages.js b/src/generic/messages.js
index fdf916a..816378a 100644
--- a/src/generic/messages.js
+++ b/src/generic/messages.js
@@ -11,6 +11,11 @@ const messages = defineMessages({
     defaultMessage: 'register',
     description: 'Text in a link, prompting the user to create an account.  Used in "learning.logistration.alert"',
   },
+  signInLowercase: {
+    id: 'learning.logistration.login', // ID left for historical purposes
+    defaultMessage: 'sign in',
+    description: 'Text in a link, prompting the user to log in.  Used in "learning.logistration.alert"',
+  },
   signInSentenceCase: {
     id: 'general.signIn.sentenceCase',
     defaultMessage: 'Sign in',
-- 
2.34.1

